signing_service/README.md

```diff
@@ -0,0 +1,68 @@
+# signing-service
+
+Stand-alone Python application used for signing Ethereum transactions received via MiddleMan protocol.
+
+
+### Install
+
+You can install this library:
+
+``` bash
+python setup.py install
+```
+
+
+### Install for development
+
+You can install this library for development:
+
+``` bash
+python setup.py develop
+```
+
+
+### Running tests
+
+You can run automated tests with:
+
+``` bash
+python -m pytest -p no:django .
```

Comment:
Please specify from which directory. Top-level or inside `signing_service`? Also, does it require the rest of Concent to work or can it be run if you just take the bits belonging to `signing_service` and `middleman_protocol` and put it in a separate directory (like we do in deployment)?

Reply:
> Please specify from which directory. Top-level or inside signing_service?

I didn't mention this because it works both ways. 

> Also, does it require the rest of Concent to work or can it be run if you just take the bits belonging to signing_service and middleman_protocol and put it in a separate directory (like we do in deployment)?

It is described as `Stand-alone` at the beginning of the file.

Reply:
> I didn't mention this because it works both ways.

Great.

> It is described as `Stand-alone` at the beginning of the file.

I mean just the tests you're running here. Are these tests completely independent of Concent's? If so, it's fine.